specification/containerservice/resource-manager/Microsoft.ContainerService/fleet/readme.python.md

```diff
@@ -1,141 +1,19 @@
-
-``` yaml $(tag) == 'package-2023-10' && $(python)
-namespace: azure.mgmt.containerservicefleet.v2023_10_15
-output-folder: $(python-sdks-folder)/containerservice/azure-mgmt-containerservicefleet/azure/mgmt/containerservicefleet/v2023_10_15
-```
-
-### Tag: package-2023-08-preview and python
-
-These settings apply only when `--tag=package-2023-08-preview --python` is specified on the command line.
-Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-
-``` yaml $(tag) == 'package-2023-08-preview' && $(python)
-namespace: azure.mgmt.containerservicefleet.v2023_08_15_preview
-output-folder: $(python-sdks-folder)/containerservice/azure-mgmt-containerservicefleet/azure/mgmt/containerservicefleet/v2023_08_15_preview
-```
-
-### Tag: package-2023-06-preview and python
-
-These settings apply only when `--tag=package-2023-06-preview --python` is specified on the command line.
-Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-
-``` yaml $(tag) == 'package-2023-06-preview' && $(python)
-namespace: azure.mgmt.containerservicefleet.v2023_06_15_preview
-output-folder: $(python-sdks-folder)/containerservice/azure-mgmt-containerservicefleet/azure/mgmt/containerservicefleet/v2023_06_15_preview
-```
-
-### Tag: package-2023-03-preview and python
-
-These settings apply only when `--tag=package-2023-03-preview --python` is specified on the command line.
-Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-
-``` yaml $(tag) == 'package-2023-03-preview' && $(python)
-namespace: azure.mgmt.containerservicefleet.v2023_03_15_preview
-output-folder: $(python-sdks-folder)/containerservice/azure-mgmt-containerservicefleet/azure/mgmt/containerservicefleet/v2023_03_15_preview
-```
-
-### Tag: package-2022-09-preview and python
-
-These settings apply only when `--tag=package-2022-09-preview --python` is specified on the command line.
-Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-
-``` yaml $(tag) == 'package-2022-06-preview' && $(python)
-namespace: azure.mgmt.containerservicefleet.v2022_09_02_preview
-output-folder: $(python-sdks-folder)/containerservice/azure-mgmt-containerservicefleet/azure/mgmt/containerservicefleet/v2022_09_02_preview
-```
-
-### Tag: package-2022-07-preview and python
-
-These settings apply only when `--tag=package-2022-07-preview --python` is specified on the command line.
-Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-
-``` yaml $(tag) == 'package-2022-07-preview' && $(python)
-namespace: azure.mgmt.containerservicefleet.v2022_07_02_preview
-output-folder: $(python-sdks-folder)/containerservice/azure-mgmt-containerservicefleet/azure/mgmt/containerservicefleet/v2022_07_02_preview
+title: ContainerServiceFleetMgmtClient
```

Comment:
Let's hold on until other packages are done.

Reply:
got it